src/main/java/mate/bookshopapp/dto/book/UpdateBookDto.java

```diff
@@ -1,11 +1,15 @@
 package mate.bookshopapp.dto.book;
 
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
+import mate.bookshopapp.validation.Isbn;
 
-public record UpdateBookDto(String title,
-                            String author,
-                            String isbn,
-                            BigDecimal price,
-                            String description,
+public record UpdateBookDto(@NotNull String title,
+                            @NotNull String author,
```

Comment:
for strings let's use NotBlank
https://www.baeldung.com/java-bean-validation-not-null-empty-blank